youtube_dl/compat.py

```diff
@@ -212,6 +213,43 @@ def data_open(self, req):
 except ImportError:  # Python 2.6
     from xml.parsers.expat import ExpatError as compat_xml_parse_error
 
+if sys.version_info[0] >= 3:
+    compat_etree_fromstring = xml.etree.ElementTree.fromstring
+else:
+    # on python 2.x the attributes and text of a node aren't always unicode
+    # objects
```

Comment:
"not always" sounds strange. Maybe it's better to explicitly state that in Python 2.x [```ElementTree._fixtext()```](https://hg.python.org/cpython/file/2.7/Lib/xml/etree/ElementTree.py#l1508) decode unicode strings to ascii if possible?